gsadjust/tab_data.py

```diff
@@ -481,14 +481,17 @@ def checkselected(self):
         """
         QtWidgets.QApplication.setOverrideCursor(QtCore.Qt.WaitCursor)
         qmodelindex = self.data_view.selectedIndexes()
+        indx2 = None
```

Comment:
If the loop below is 0-length (skipped) then you can get to the following line without indx2 being set. I added `None` as a default here and check below to skip the `.setData` in this case.